### ISSUE TITLE
JupyterLab/Lumino Accessibility Demo Branches

### ISSUE THREAD

**Issue by m158261** [t03857785/jupyterlab#12]
Linked JupyterLab and Lumino Binder Instance - [![Binder](https://mybinder.org/badge_logo.svg)](https://mybinder.org/v2/gist/m158261/7232671f04a5de8f1ce4b810f5ea8a3a/HEAD?urlpath=lab)

[Public Gist]( https://gist.github.com/m158261/7232671f04a5de8f1ce4b810f5ea8a3a)

**JupyterLab**

 - Input fields in dialogs need to be labeled #14560
 - Modal dalogs are not accessible (mplement dialog element for dialogs with accessibility) #14560
 - Add aria roles and labels #9622
 - added dynamic functionality to the text spacing #14561
 - Added title to close tab icon #14973
 - 400% Zoom Screen usebility enhancements - updated #14766
 - Make Status bar accessible at 400% zoom #14854
 - Fix skiplink and add placeholder for additional skiplinks (Linked with PR #14575) #14597
 - Clarify labels for user inputs and link dangling labels #15222
 - Ensure consistent link underline and color on hover #15181
 - Addition of alt descriptions #15265
 - Full Low Vision Support (400% zoom) (Mobile/Tablet-Responsive Support) #14766
 - Visited link are invisible in light theme #15406
 - Change close launcher div to button for screen reader accessibility #15347
 
**Lumino**

 - Added "tabindex=0" for sidebar accessiblity #583
 - Added role of button to addButton to make it keyboard accessbile when using screenreader #615
 - elements respond to enter or spacebar #590
 - Close launcher button styling and close launcher with delete key #655

**Comment by tonyfast** [comment]
i couldnt load notebook on the gesis. im trying to build on ovh now

https://ovh.mybinder.org/v2/gist/m158261/7232671f04a5de8f1ce4b810f5ea8a3a/HEAD?urlpath=lab
